beacon-chain/db/filesystem/blob.go

```diff
@@ -234,6 +240,8 @@ func (bs *BlobStorage) Prune(currentSlot primitives.Slot) error {
 			if err := bs.processFolder(folder, currentSlot, retentionSlots); err != nil {
 				return err
 			}
+			blobsPrunedCounter.Inc()
+			blobsTotalGauge.Dec()
```

Comment:
This doesn't properly account for the number of blobs that were pruned when a directory is deleted.